### PR TITLE
Homework01

### PR DESCRIPTION
**Pull request by njie-DP**
@whitneykasenetz @sejalkesh
Homework01 is ready for peer review. Thanks in advance. :)

### CODE REVIEW

File: participation/hw01/README.md [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
----
-
-```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
-```
+# Ni's Homework Page
+|**Date**|**2021/01/25**|
+|--------|--------------|
 
 This is the repository for hosting and posting my homework assignments for Programming with Data, please feel free to comment on my codes.
```
Comment (whitneykasenetz) [inline participation/hw01/README.md:6]: I unwittingly skipped over your personal introduction when I displayed the rich diff but I found it when reviewing your code in the source diff. Although it is a little sneaky I am glad you showed me something new.

Comment (whitneykasenetz) [inline participation/hw01/README.md:6]: You may lose points for not including information about how to navigate the repository.

File: participation/hw01/README.md [hunk 2]
```diff
@@ -25,7 +17,5 @@ My name is Ni, a third-year graduate student from the University of South Florid
 I also want to introduce you my two lovely Siamese cats!  
 ![Hulu & Pulu](https://static.wixstatic.com/media/41c14b_0dc6344d51a245a2a5046849dd86b1bf~mv2.jpg)
```
Comment (whitneykasenetz) [inline participation/hw01/README.md:19]: Other than missing navigation instructions, it looks like you followed all the other instructions. Good work!

File: participation/hw01/Exploring bfi.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    ##  Max.   :6.000   Max.   :6.00   Max.   :6.000   Max.   :6.000   Max.   :6.000  
+    ##  NA's   :36      NA's   :29     NA's   :22                      NA's   :28     
+    ##        O4              O5           gender        education         age       
+    ##  Min.   :1.000   Min.   :1.00   Min.   :1.000   Min.   :1.00   Min.   : 3.00  
+    ##  1st Qu.:4.000   1st Qu.:1.00   1st Qu.:1.000   1st Qu.:3.00   1st Qu.:20.00  
+    ##  Median :5.000   Median :2.00   Median :2.000   Median :3.00   Median :26.00  
+    ##  Mean   :4.892   Mean   :2.49   Mean   :1.672   Mean   :3.19   Mean   :28.78  
+    ##  3rd Qu.:6.000   3rd Qu.:3.00   3rd Qu.:2.000   3rd Qu.:4.00   3rd Qu.:35.00  
+    ##  Max.   :6.000   Max.   :6.00   Max.   :2.000   Max.   :5.00   Max.   :86.00  
+    ##  NA's   :14      NA's   :20                     NA's   :223
```
Comment (whitneykasenetz) [inline participation/hw01/Exploring bfi.md:174]: It looks to me like you followed all the instructions for this assignment. I had no problem reading your work and understanding what you were doing. Good job, Ni!

File: README.md [hunk 4]
```diff
@@ -15,7 +16,5 @@ My name is Ni, a third-year graduate student from the University of South Florid
 I also want to introduce you my two lovely Siamese cats!  
 ![Hulu & Pulu](https://static.wixstatic.com/media/41c14b_0dc6344d51a245a2a5046849dd86b1bf~mv2.jpg)
```
Comment (sejalkesh) [inline README.md:18]: I tried to run this image but I got an error.

File: participation/hw01/Exploring bfi.md (hunk 3 above)
Comment (sejalkesh) [reply participation/hw01/Exploring bfi.md:174]: You did a great job, Ni! Your code was easy to read and very organized. I had no trouble understanding the conclusions made from the data set.

File: participation/hw01/Exploring bfi.Rmd [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+title: "Exploring bfi"
+author: "Ni Jie"
+date: "1/25/2021"
+output: html_document
+---
+
+## Exploring bfi dataset
+
+I explored the "bfi" dataset from "psych" package. The variables are BFI items, gender, education, and age. This dataset includes 2800 observations and 28 variables. The mean age of participants is 28.78 with a standard deviation of 11.13.
```
Comment (sejalkesh) [inline participation/hw01/Exploring bfi.Rmd:10]: You did a great job of explaining what the data in this specific package means.

File: participation/hw01/Exploring bfi.md (hunk 3 above)
Comment (njie-DP) [reply participation/hw01/Exploring bfi.md:174]: Thank you both!

**Pull request closed without merging**